### ISSUE TITLE
403 forbidden on wat.tv

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MiraculousLegendas/get-flash-videos#97]
```
I tried the one in the testing suite and then a few more, they all return 
403.

studio@studio-laptop:~/get-flash-videos$ ./get_flash_videos --debug 
http://www.wat.tv/video/disturbia-rihanna-tuerie-ete-sa5h_mrz0_.html
1 plugin installed:
- Hulu.pm
Downloading http://www.wat.tv/video/disturbia-rihanna-tuerie-ete-
sa5h_mrz0_.html
-> GET http://www.wat.tv/video/disturbia-rihanna-tuerie-ete-sa5h_mrz0_.html
<- 200 text/html (156146)
Trying to open plugin ~/.get_flash_videos/plugins/Www.pm
Trying to open plugin ~/.get_flash_videos/plugins/Wat.pm
Using method 'wat' for http://www.wat.tv/video/disturbia-rihanna-tuerie-
ete-sa5h_mrz0_.html
-> GET http://www.wat.tv/interface/contentv2/1319525
<- 200 application/json; charset="utf-8" (21053)
Downloading 
http://www.wat.tv/get/ce379db41c20c6c50fc3bc52ebc27076/4bbe45d4/2/FLV-
320x240/95/25/1319525.flv...
-> GET 
http://www.wat.tv/get/ce379db41c20c6c50fc3bc52ebc27076/4bbe45d4/2/FLV-
320x240/95/25/1319525.flv
<- 403 text/html (0)
Couldn't download 
http://www.wat.tv/get/ce379db41c20c6c50fc3bc52ebc27076/4bbe45d4/2/FLV-
320x240/95/25/1319525.flv: 403 Not authorized
Couldn't download any videos.
studio@studio-laptop:~/get-flash-videos$ ./get_flash_videos 
http://www.wat.tv/video/coiffure-en-video-realiser-11q42_11bz8_.html
Downloading http://www.wat.tv/video/coiffure-en-video-realiser-
11q42_11bz8_.html
Using method 'wat' for http://www.wat.tv/video/coiffure-en-video-realiser-
11q42_11bz8_.html
Downloading 
http://www.wat.tv/get/99eff32ec476e72d730750f2b952090e/4bbe447e/2/FLV-
320x240/01/14/1760114.flv...
Couldn't download 
http://www.wat.tv/get/99eff32ec476e72d730750f2b952090e/4bbe447e/2/FLV-
320x240/01/14/1760114.flv: 403 Not authorized
Couldn't download any videos.
studio@studio-laptop:~/get-flash-videos$ ./get_flash_videos 
http://www.wat.tv/video/casting-89-1umji_1blbn_.html
Downloading http://www.wat.tv/video/casting-89-1umji_1blbn_.html
Using method 'wat' for http://www.wat.tv/video/casting-89-1umji_1blbn_.html
Downloading 
http://www.wat.tv/get/19beb33539242dee14822c1572f6d24a/4bbe40fa/2/FLV-
320x240/85/10/3108510.flv...
Couldn't download 
http://www.wat.tv/get/19beb33539242dee14822c1572f6d24a/4bbe40fa/2/FLV-
320x240/85/10/3108510.flv: 403 Not authorized
Couldn't download any videos.

```

Original issue reported on code.google.com by `tryagain...@gmail.com` on 8 Apr 2010 at 9:17

**Comment by GoogleCodeExporter** [comment]
```
Same problem on Mac OS X and Debian 5.0.
```

Original comment by `after...@gmail.com` on 10 Apr 2010 at 6:07

**Comment by GoogleCodeExporter** [comment]
```
Fixed in git 0318ef1.
```

Original comment by `zakflash...@gmail.com` on 21 Apr 2010 at 5:28
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
```
I also can not download wat.tv use get_flash_videos, how to fix it? Thank's.
```

Original comment by `rsrudysu...@gmail.com` on 10 Sep 2010 at 2:22
